spec/unit/pool_manager_spec.rb

```diff
@@ -46,6 +46,30 @@
     end
   end
 
+  describe '#load_used_providers' do
+    let(:config) { YAML.load(<<-EOT
+---
+:config:
+:providers:
+  :mock:
+:pools:
+  - name: '#{pool}'
+    size: 1
+    provider: 'spoof'
+    EOT
+    )
+    }
+    it do
```

Comment:
nit: Would prefer a descriptive name for test

Reply:
+1 to descriptive name for test, otherwise looks ready to roll.